index.js

```diff
@@ -98,6 +142,9 @@ function myMap(array, callback) {
 }
 
 // ***** Callbacks - Question 1 *****
+function triple(arrlenght){
+  return arrlenght * 3
+}
```

Comment:
Good! Only thing is your parameter name `arrlenght` indicates that maybe this function takes in the length of an array as an argument. However, it should take in any number value. `myMap` will call this function with each element in an array (not the array's length). It's just semantics, but maybe a better name for the parameter here would be `num` or `element`, etc.